src/main/scala/strawman/collection/mutable/Seq.scala

```diff
@@ -0,0 +1,76 @@
+package strawman
+package collection
+package mutable
+
+trait Seq[A] extends strawman.collection.Seq[A] with Growable[A] {
```

Comment:
Yes, good point. Current collections ignore this issue. Do people have proposals how to solve this? It seems there's no nice option.